### DISCUSSION TITLE
Setting up run check

### DISCUSSION THREAD

**Discussion by ghost** [upptime/upptime#734] [Q&A] [answered]
Hi, I need to set up check every 5 minutes for all sites. I see this
schedule:
    - cron: "0 1 * * *"
how to configure so that upptime runs checks of all sites every 5 minutes and writes to the git history that added to the list of sites in the configuration file

**Comment by AnandChowdhary** [comment, marked as answer]
You should update the schedule for graphs and response time to update the graphs and commits every 5 min respectively:

```
workflowSchedule:
  graphs: "0 0 * * *"
  responseTime: "0 23 * * *"
```
